### PR TITLE
Udq rst preparations

### PR DESCRIPTION
**Pull request by joakim-hove**
Some early stage - quite trivial preparations for UDQ restart.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
Looks good for the most part, but I have one naming suggestion concerning multisegment wells.

File: opm/io/eclipse/rst/state.hpp [hunk 1]
```diff
@@ -93,6 +58,24 @@ struct RstState {
...
+
+    void add_wells(const std::vector<std::string>& zwel,
+                   const std::vector<int>& iwel,
+                   const std::vector<float>& swel,
+                   const std::vector<double>& xwel,
+                   const std::vector<int>& icon,
+                   const std::vector<float>& scon,
+                   const std::vector<double>& xcon);
+
+    void add_msw_wells(const std::vector<std::string>& zwel,
```
[the remark this answers is not in the document]
Comment (joakim-hove) [reply opm/io/eclipse/rst/state.hpp]: OK - name changed to `add_msw()`

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**